packages/rum/src/domain/record/serialization/serializationScope.ts

```diff
@@ -0,0 +1,9 @@
+import type { NodeIds } from '../nodeIds'
+
+export interface SerializationScope {
+  nodeIds: NodeIds
+}
+
```

Comment:
Question: just for my understanding, what are you planning to add in the `scope`?